Add `Wiki maintainers` page (#25)
osuTitanic/wiki: wiki/People/Wiki_maintainers/en.md

## Wiki maintainers

> Wiki maintainers are a group of users who are in charge of maintaining the wiki and merging changes into the official Titanic! wiki.
> 
> ## Roles and responsibilities
> Wiki maintainers are responsible for reviewing and merge pull requests to make sure there are no errors.

Reviewer: ```suggestion
Wiki maintainers are a group of users who are in charge of maintaining the wiki and merging changes into the official Titanic! wiki. They are responsible for reviewing and merge pull requests to make sure there are no errors.
```
What do you think about this?

Reviewer 2: looks better in that way though.

Author: Sounds way better (What the fuck was I doing at like 1 am)

> ## Current Members

Reviewer 2: ```suggestion
## Current members
```

User | Added Date 
---|---

> [BlueChinchompa](https://osu.titanic.sh/u/40)   | Since The Beginning 

Reviewer 2: ```suggestion
[BlueChinchompa](https://osu.titanic.sh/u/40) | Since The Beginning 
```
